application/aws/aws.go

```diff
@@ -0,0 +1,154 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/gobuffalo/envy"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+const AwsS3RegionEnv = "AWS_REGION"
+const AwsS3EndpointEnv = "AWS_S3_ENDPOINT"
+const AwsS3DisableSSLEnv = "AWS_S3_DISABLE_SSL"
+const AwsS3BucketEnv = "AWS_S3_BUCKET"
+const AwsS3AccessKeyIDEnv = "AWS_S3_ACCESS_KEY_ID"
+const AwsS3SecretAccessKeyEnv = "AWS_S3_SECRET_ACCESS_KEY"
+
+type ObjectUrl struct {
+	Url        string
+	Expiration time.Time
+}
+
+type awsConfig struct {
+	awsAccessKeyID     string
+	awsSecretAccessKey string
+	awsEndpoint        string
+	awsRegion          string
+	awsS3Bucket        string
+	awsDisableSSL      bool
+	getPresignedUrl    bool
+}
+
+// presigned URL expiration
+const urlLifespan = 10 * time.Minute
+
+// String returns a pointer to the string value passed in.
+func String(v string) *string {
+	return &v
+}
+
+func GetS3ConfigFromEnv() awsConfig {
+	var a awsConfig
+	a.awsAccessKeyID = envy.Get(AwsS3AccessKeyIDEnv, "")
+	a.awsSecretAccessKey = envy.Get(AwsS3SecretAccessKeyEnv, "")
+	a.awsEndpoint = envy.Get(AwsS3EndpointEnv, "")
+	a.awsRegion = envy.Get(AwsS3RegionEnv, "")
+	a.awsS3Bucket = envy.Get(AwsS3BucketEnv, "")
+
+	if disableSSL, err := strconv.ParseBool(envy.Get(AwsS3DisableSSLEnv, "false")); err == nil {
+		a.awsDisableSSL = disableSSL
+	} else {
```

Comment:
The `else` block is redundant, right, i.e. the initial value of a bool is already false? Or, do you maybe want to deal with the error?